docs/src/components/landing-studio/Hero.js

```diff
@@ -0,0 +1,295 @@
+import * as React from 'react';
```

Comment:
I assume this is just a moved file, lmk if I should review it.

Reply:
Yep, this is a moved file